docs/contributing.md

```diff
@@ -0,0 +1,12 @@
+# Contributing
+Contributions are welcome. We have a few asks of any new contributors.
```

Comment:
```suggestion
Contributions are welcome. We have a few requests of any new contributors.
```
Maybe the noun "ask" is more common in circles I don't frequent, but I think "request" is more appropriate for this context.